### ISSUE TITLE
requestCurrentLocation Implementation is not available

### ISSUE THREAD

**Issue by MohanOnsurity** [Albrodiaz/LocationComposeExample#1]

**Comment by Albrodiaz** [comment]
Exactly, that function is not implemented because it was not necessary for this example. If you want to expand the functionality, you can implement it yourself to know the user's last known location.

**Comment by MohanOnsurity** [comment]
I have implemented it, I will raise PR for the same.
thanks @Albrodiaz
